ui/src/components/Event/SendEventModal.tsx

```diff
@@ -175,8 +169,8 @@ export const SendEventModal = () => {
             }}
             options={{
               fixedOverflowWidgets: true,
-              formatOnPaste: true,
-              formatOnType: true,
+              formatOnPaste: false,
+              formatOnType: false,
```

Comment:
Oh interesting - I only noticed this last night when I removed `input` from `value={}`. It wasn't working before me, I don't mind this either way though.